internal/models/reservation_model.go

```diff
@@ -59,6 +59,9 @@ type AWSDetail struct {
 
 	// Immediately power off the system after initialization
 	PowerOff bool `json:"poweroff"`
+
+	// PubkeyName on AWS in given region. Found by the EnsurePubkey job.
+	PubkeyNameAWS string `json:"pubkey_name"`
```

Comment:
I suggest to be consistent:

```suggestion
	PubkeyName string `json:"pubkey_name"`
```

Reply:
Done